generic/ras_extended.py

```diff
@@ -173,8 +173,23 @@ def test1_vpdupdate(self):
                                         "head -1", shell=True).strip()
         if path_db:
             copyfile_path = os.path.join(self.outputdir, 'vpd.db')
-            copyfile(path_db, copyfile_path)
+            shutil.copyfile(path_db, copyfile_path)
             self.run_cmd("vpdupdate --path=%s" % copyfile_path)
+        if os.path.exists('/var/lib/lsvpd/run.vpdupdate'):
+            path = '/var/lib/lsvpd/run.vpdupdate'
+        elif os.path.exists('/var/lib/lsvpd/run.vpdupdate'):
+            path = '/run/run.vpdupdate'
+        shutil.move(path, "/root/run.vpdupdate")
+        self.log.info("Running vpdupdate after removing run.vpdupdate")
+        self.run_cmd("vpdupdate")
+        shutil.move("/root/run.vpdupdate", path)
```

Comment:
Many instances having files names hardcoded and the same used in different places. It's a good practice to use variable names.